baseapp-chats/baseapp_chats/graphql/subscriptions.py

```
@@ -6,41 +6,51 @@
-        return ChatRoomOnRoomUpdate(room=Edge(node=room))
+    @classmethod
+    def new_message(cls, message):
+        cls.room_updated(message.room)
 
     @classmethod
-    def new_message(cls, message, **kwargs):
-        cls.broadcast(
-            group="chat",
-            payload={"message": message},
-        )
+    def room_updated(cls, room, removed_participants=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)